### DISCUSSION TITLE
some questions about the customs fonts

### DISCUSSION THREAD

**Discussion by BuzzLEquerre** [flareteam/flare-game#981] [General]
Hi guys, I have some questions about font settings. Because I would like to use the Baldur's Gate 1 font "sherwood" only I would like not to apply it for dialogues and descriptions.

For this I replaced the "LiberationSans-Regular.ttf" with the "font_regular" and I created a new style, "font_classic" which uses the regular.

- The problem is that I can't find the file or I don't know the function to apply the "font_classic" to the "topic" of the dialogues.
![Capture d'écran 2024-12-13 212309](https://github.com/user-attachments/assets/6ee59e26-41d4-47e1-92bc-95b790aa28bf)
- This is the same for the combat stats of enemies and npc.
![Capture d'écran 2024-12-13 212441](https://github.com/user-attachments/assets/e605cc0a-0f0b-40bc-970b-f7d5b6fd1d6d)

If anyone has an idea how to solve my problem I'm interested :)

**Comment by dorkster** [comment]
For NPC dialog, `font_who=font_classic` and `font_dialog=font_classic` would be the intended way to do what you want. However, I think there's an oversight with regards to the dialog choices. Because they are technically buttons, they just use `font_regular`. Shouldn't be hard to fix that.

Combat text is another oversight. There simply isn't a property to assign a font, so it just uses `font_regular` by default.
